webpack/production.config.js

```diff
@@ -0,0 +1,3 @@
+const config = require('./../webpack.config');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.